app/backend/app.py

```diff
@@ -79,7 +80,10 @@ def content_file(path):
     mime_type = blob.properties["content_settings"]["content_type"]
```

Comment:
blob.properties["content_settings"] could be None or raise a KeyError, this isn't being handled at the moment.

Reply:
I've added a guard that 404s on the lines before. I looked through the SDK code and docs and I must admit, it's not clear to me which of these could be None/empty, but my guard seems reasonable, and VS Code is no longer concerned about it.